www/module.config.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
     dist: The sub-directory of the source code where it puts its build artifact.  Often "dist".
```

Comment:
do we need to gitignore this one too? Seems like it won't work unless you are edx

Reply:
uuuuuugh. so, we need the part that links to the platform repo.  I think this is a place where we should use this to alias to another locally-checked-out repo in the www package.json.  I will poke at.